flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java

```diff
@@ -20,34 +20,29 @@
 
 import org.apache.flink.annotation.PublicEvolving;
 import org.apache.flink.api.common.ExecutionConfig;
-import org.apache.flink.api.common.typeinfo.TypeInformation;
-import org.apache.flink.api.common.typeutils.TypeSerializer;
-import org.apache.flink.api.java.typeutils.TypeExtractor;
-import org.apache.flink.core.memory.DataInputViewStreamWrapper;
-import org.apache.flink.core.memory.DataOutputViewStreamWrapper;
 import org.apache.flink.util.Preconditions;
 
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
 import java.io.Serializable;
 
 import static java.util.Objects.requireNonNull;
 
 /**
- * Base class for state descriptors. A {@code StateDescriptor} is used for creating partitioned
- * {@link State} in stateful operations. This contains the name and can create an actual state
- * object given a {@link StateBackend} using {@link #bind(StateBackend)}.
- *
+ * Base class for state descriptors. A {@code StateDescriptor} is used for creating
+ * {@link State keyed state} in stateful operations. The descriptor contains the name of the state,
```

Comment:
Just a quick note: `StateDescriptors` are also used for non-keyed state, i.e. operator state.